hack/tests/subcommand-scorecard.sh

```diff
@@ -48,6 +51,8 @@ then
 	exit 1
 fi
 
+set -xe 
```

Comment:
Is there a way to refactor this such that we can leave `set -xe` enabled even while expecting this command to exit non-zero? Maybe something similar to the `if` statement in my other comment?

That way if some command fails that we don't expect to fail, then we'll still fail the test.